src/pages/Lobby/components/PlayersTable/PlayersTable.tsx

```diff
@@ -0,0 +1,24 @@
+import { Position, Row } from './styles';
+import { PlayersTableProps } from './types';
+
+export function PlayersTable({ players }: PlayersTableProps) {
+  function getPlayerNameItem(current: boolean, name: string) {
+    if (!current) {
+      return name;
+    }
+    return 'Você';
+  }
```

Comment:
**Suggestion:** this function can be defined outside the component as it does not need any state or prop